diff -up tiff-4.3.0/Makefile.am.1~ tiff-4.3.0/Makefile.am
--- tiff-4.3.0/Makefile.am.1~	2021-03-07 12:32:24.000000000 +0100
+++ tiff-4.3.0/Makefile.am	2021-04-17 03:11:22.651296572 +0200
@@ -57,7 +57,7 @@ distcheck-hook:
 	rm -rf $(distdir)/_build/cmake
 	rm -rf $(distdir)/_inst/cmake
 
-SUBDIRS = port libtiff tools build contrib test man html
+SUBDIRS = port libtiff tools build test man html
 
 release:
 	(rm -f $(top_srcdir)/RELEASE-DATE && echo $(LIBTIFF_RELEASE_DATE) > $(top_srcdir)/RELEASE-DATE)
diff -up tiff-4.3.0/Makefile.in.1~ tiff-4.3.0/Makefile.in
--- tiff-4.3.0/Makefile.in.1~	2021-04-17 03:11:22.651296572 +0200
+++ tiff-4.3.0/Makefile.in	2021-04-17 03:11:52.634642163 +0200
@@ -434,7 +434,7 @@ EXTRA_DIST = \
 	libtiff-4.pc.in
 
 dist_doc_DATA = $(docfiles)
-SUBDIRS = port libtiff tools build contrib test man html
+SUBDIRS = port libtiff tools build test man html
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libtiff-4.pc
 
