diff -Naur tiff-4.0.6/Makefile.am tiff-4.0.6.tpg/Makefile.am
--- tiff-4.0.6/Makefile.am	2015-09-06 19:30:46.000000000 +0000
+++ tiff-4.0.6.tpg/Makefile.am	2016-03-14 21:38:18.300007836 +0000
@@ -61,7 +61,7 @@
 	rm -rf $(distdir)/_build/cmake
 	rm -rf $(distdir)/_inst/cmake
 
-SUBDIRS = port libtiff tools build contrib test man html
+SUBDIRS = port libtiff tools build test man html
 
 release:
 	(rm -f $(top_srcdir)/RELEASE-DATE && echo $(LIBTIFF_RELEASE_DATE) > $(top_srcdir)/RELEASE-DATE)
diff -Naur tiff-4.0.6/Makefile.in tiff-4.0.6.tpg/Makefile.in
--- tiff-4.0.6/Makefile.in	2015-09-12 19:48:44.000000000 +0000
+++ tiff-4.0.6.tpg/Makefile.in	2016-03-14 21:38:32.640386664 +0000
@@ -435,7 +435,7 @@
 	nmake.opt
 
 dist_doc_DATA = $(docfiles)
-SUBDIRS = port libtiff tools build contrib test man html
+SUBDIRS = port libtiff tools build test man html
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libtiff-4.pc
 
